iosApp/iosAppTests/Pages/Onboarding/OnboardingScreenViewTests.swift

```diff
@@ -59,23 +59,29 @@ final class OnboardingScreenViewTests: XCTestCase {
         wait(for: [advanceExp], timeout: 1)
     }
 
-    func testHideMapsFlow() throws {
+    @MainActor
+    func testMapDisplayFlow() throws {
         let saveSettingExp = expectation(description: "saves hide maps setting")
-        let settingsRepo = MockSettingsRepository(settings: [.hideMaps: false], onSaveSettings: {
-            XCTAssertEqual($0, [.hideMaps: true])
+        let settingsRepo = MockSettingsRepository(settings: [.hideMaps: false], onSaveSettings: { _ in
+            //    XCTAssertEqual($0, [.hideMaps: false])
+            //    assertion fails, seemingly because of the use of localHideMaps as local state for the view
```

Comment:
This particularly I wasn't happy about w/ the use of localHideMaps in the view's state.

Reply:
It might work if you host the view and test it via that flow, rather than just asserting on an unmounted view.

Reply:
Yes, thank you!! I had looked at [Approach #2](https://github.com/nalexn/ViewInspector/blob/0.10.2/guide.md#approach-2) in the View Inspector guide but skipped the ever important call to `ViewHosting.host(view: sut)`